ruby/ext/google/protobuf_c/message.c

```diff
@@ -248,16 +260,26 @@ int Message_initialize_kwarg(VALUE key, VALUE val, VALUE _self) {
     Map_merge_into_self(map, val);
   } else if (upb_fielddef_label(f) == UPB_LABEL_REPEATED) {
     VALUE ary;
+    VALUE entry;
```

Comment:
This is only used inside the for loop, so please declare it there.

Reply:
Done!